compute/instances.go

```
@@ -364,7 +367,10 @@ func (c *InstancesClient) GetInstance(input *GetInstanceInput) (*InstanceInfo, e
 	}
 	responseBody.SSHKeys = sshKeyNames
 
-	responseBody.Networking = c.unqualifyNetworking(responseBody.Networking)
+	responseBody.Networking, err = c.unqualifyNetworking(responseBody.Networking)
```

And define a new var, `networkingErr` so as not to mask any errors from above